flask_ask/models.py

```diff
@@ -440,6 +440,25 @@ def clear_queue(self, stop=False):
 
         self._response['directives'].append(directive)
         return self
+    
+def progressive_response(speech:str):
+    """Causes Alexa to speak before your skill sends its full response."""
+    response = {
+        "header":{
+            "requestId":"amzn1.echo-api.request.xxxxxxx"
+        },
+        "directive":{
+            "type":"VoicePlayer.Speak",
+            "speech":"This text is spoken while your skill processes the full response."
+        }
+    }
+    response['header']['requestId'] = context.System.apiAccessToken
+    response['directive']['speech'] = request.requestId
+
+
+    headers = {"Authorization":'Bearer '+authToken, "Content-Type":"application/json"}
+    r = requests.post("https://api.amazonalexa.com/v1/directives", headers=headers, json=response)
```

Comment:
This URL is region specific, for me this URL is not working,

This URL "https://api.eu.amazonalexa.com/v1/directives" is working for me.

Reply:
Thanks!  It is no longer region specific.